package.json

```diff
@@ -48,10 +48,10 @@
     "babel-runtime": "=5.8.24",
     "bluebird": "^2.9.25",
     "date-utils": "^1.2.18",
-    "lodash": "^3.9.1",
+    "lodash": "^4.6.1",
     "mobile-json-wire-protocol": "^1.3.2",
     "npmlog": "^2.0.1",
-    "request-promise": "^1.0.2",
+    "request-promise": "^2.0.1",
```

Comment:
I don't think we should do this until we are ready to upgrade bluebird throughout all the packages.